gobotics-frontend/src/containers/AssetList.tsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux'
+import Asset from "../ducks/assets/types";
+import AssetList from '../components/AssetList'
+
+const getAssets = (assets: Asset[], filter = "ALL") => {
+    switch(filter) {
+        // can filter down the list by type here maybe?
+        default:
+            return assets;
+    }
+}
+const mapStateToProps = state => {
+    return {
+      todos: getAssets(state.assets)
+    }
+  }
```

Comment:
How should I define `state` type here? ( Parameter 'state' implicitly has an 'any' type.  TS7006 )

Reply:
Also should the type def live here or in the ducks folder?